tests/auto_inject/test_auto_inject_install.py

```diff
@@ -129,9 +130,17 @@ def test_crash_no_zombie(self):
             raise
 
         # At this point, there should be no zombies and no child pids
+        # but we apply a retry policy due to the app can take time to crash
         child_pids = get_child_pids(virtual_machine).strip()
+        for _attempt in range(5):
+            child_pids = get_child_pids(virtual_machine).strip()
```

Comment:
Minor: The first call is ignored? I don't do a lot of python, but I believe you don't need to declare `child_pids` outside the `for`

Reply:
uppss, your are right. Fixed. Thanks!